pkg/storage/sqlstorage/accounts_test.go

```
@@ -0,0 +1,81 @@
+package sqlstorage
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/numary/ledger/pkg/storage"
+	"github.com/pborman/uuid"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAccounts(t *testing.T) {
+	d := NewDriver("sqlite", &sqliteDB{
+		directory: os.TempDir(),
+		dbName:    uuid.New(),
+	})
+
+	assert.NoError(t, d.Initialize(context.Background()))
+
+	defer func(d *Driver, ctx context.Context) {
+		assert.NoError(t, d.Close(ctx))
+	}(d, context.Background())
+
+	store, _, err := d.GetStore(context.Background(), "foo", true)
+	assert.NoError(t, err)
+
+	_, err = store.Initialize(context.Background())
+	assert.NoError(t, err)
+
+	q := storage.Query{
+		Params: map[string]interface{}{
+			"balance": "toto",
+		},
+	}
```

should be moved into the sub test `panic invalid balance`